system/src/Grav/Common/File/CompiledFile.php

```diff
@@ -14,6 +15,8 @@
  */
 trait CompiledFile
```

Comment:
CompiledFile is not meant to be using locator; instead I would make static function to initialize the cache path.